pandas/_libs/tslibs/__init__.py

```diff
@@ -8,3 +8,4 @@
 from .timestamps import Timestamp
 from .timedeltas import delta_to_nanoseconds, ints_to_pytimedelta, Timedelta
 from .tzconversion import tz_convert_single
+from .c_timestamp import maybe_integer_op_deprecated # isort:skip
```

Comment:
do we really need this function here? this should not be exposed to the rest of the codebase (or directly imported if it actually is needed)

Reply:
maybe just expose it in timestamp.pyx itself to make it easier to import

Reply:
It was exposed previously, and is only used in `pandas/core/arrays/datetimelike.py`.  `_Timestamp` itself makes use of it, so it has to be in `c_timestamp.pyx` otherwise we would have a circular import.

I can change the import in `datetimelike.py` to directly import it if you want.

Reply:
sorry made the comment in the wrong place. yes just import it directly in ``dateimelike.py`` and don't put it in ``tslibs/__init__.py``